test: remove unused helper functions from test_views.py

raju249 opened the pull request:
## Description

Resolves  https://github.com/openedx/edx-platform/issues/31696

Deletes two unused helper functions from `lms/djangoapps/teams/tests/test_views.py`.

## Supporting information

NA

## Testing instructions

Running `lms/djangoapps/teams/tests/test_views.py` should pass as expected before this change.

## Deadline

None

## Other information

NA

Commit:
test: remove unused helper functions from test_views.py

Since they were not used anywhere, it is ideal to delete them.

It would be redundant to keep them.

openedx-webhooks commented:
Thanks for the pull request, @raju249!  Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.




<!-- comment:external_pr -->
<!-- jenkins ok to test -->
<!-- data: eyJkcmFmdCI6IGZhbHNlfQ== -->

e0d commented:
@raju249 We need to have a CLA on file for you before we can accept this contribution, take a look at the instructions above.

Also, I notice there are some commit-lint failures.  Please note that we use conventional commits across Open edX projects.  You can read about the details [here](https://open-edx-proposals.readthedocs.io/en/latest/best-practices/oep-0051-bp-conventional-commits.html#type).  Can you please amend your commit messages to follow our standard?

raju249 commented:
Thanks @e0d 

Just fixing my commit messages. Regarding CLA, I am awaiting a reply from the team, I have signed on my side. Over email conversation, I got to know that the legal team member is on leave for this week...

I am fine with keeping this and the other PR open until its complete. 👍

raju249 commented:
Ok, I think I messed up with merge and rebase. Let me rectify.

raju249 commented:
@e0d - I have fixed the commit messages from the direction mentioned [here](https://open-edx-proposals.readthedocs.io/en/latest/best-practices/oep-0051-bp-conventional-commits.html).

I have also rebased to latest master. I also just got a confirmation on email that the legal team member is back at work and will take care of the CLA today.

raju249 commented:
@e0d - CLA is complete. Can you or someone else please help me with a review on this PR?

raju249 commented:
@e0d - can you help re-run the pipeline, please?

e0d committed:
Merge branch 'master' into 31696-unused-helper-functions

e0d commented:
@raju249 I've pulled in changes from master as a merge commit which will update your branch and cause the tests to be re-run.

raju249 commented:
Thanks @e0d 

@feanil / @nedbat - Mind taking a look at this PR, please?

feanil reviewed (approved):
Yep, looks like these aren't used in platform testing anywhere.

feanil commented:
I'll coordinate merging.

openedx-webhooks commented:
@raju249 🎉 Your pull request was merged!  Please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/edx-platform&entry.867055334=https://github.com/openedx/edx-platform/pull/32559&entry.1484655318=https://github.com/raju249&entry.752974735=2023-06-23+09:28&entry.1917517419=2023-07-03+14:36&entry.2133058324=Yes) so we can improve your experience in the future.

<!-- comment:end_survey -->
